### PR TITLE
Docs/typo

### PR DESCRIPTION
**Pull request by zhijianma**
---
name: Pull Request
about: Create a pull request
---
## Description
- change `openai` to `openai_chat` in tutorials and examples.
- add `model_configs_template` to example
- add `example_template`
- fix some typo and filename in scripts
- fix some bugs in models 

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,21 +65,21 @@ applications in a centralized programming manner for streamlined development.
...
-|                        | Generation      | [`OllamaGenerationWrapper`](https://github.com/modelscope/agentscope/blob/main/src/agentscope/models/ollama_model.py)           | [link](https://modelscope.github.io/agentscope/en/tutorial/203-model.html#ollama-api)       | llama2, Mistral, ...                         |
-| Post Request based API | -               | [`PostAPIModelWrapper`](https://github.com/modelscope/agentscope/blob/main/src/agentscope/models/post_model.py)                 | [link](https://modelscope.github.io/agentscope/en/tutorial/203-model.html#post-request-api) | -                                            |
+| OpenAI API             | Chat            | [`OpenAIChatWrapper`](https://github.com/modelscope/agentscope/blob/main/src/agentscope/models/openai_model.py)                 | [link](https://github.com/modelscope/agentscope/blob/main/examples/model_configs_template/openai_chat_template.json)       | gpt-4, gpt-3.5-turbo, ...                    |
```
[the remark this answers is not in the document]
Comment (DavdGao) [reply README.md:70]: TBD

File: .gitignore [hunk 2]
```diff
@@ -137,4 +137,6 @@ dmypy.json
 docs/sphinx_doc/build/
 
 # Used to save loggings and files
-runs/
+*runs/
+agentscope.db
+tmp*.json
```
[the remark this answers is not in the document]
Comment (zhijianma) [reply .gitignore:142]: These `tmp*json`  files are  generated by tests.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhijianma) [reply README.md:70]: How about keep link and tempate ? 
image

[the remark this answers is not in the document]
Comment (DavdGao) [reply README.md:70]: Maybe `guide` rather than `link`?

[the remark this answers is not in the document]
Comment (zhijianma) [reply README.md]: How aboult this?
image

**Review by DavdGao — APPROVED** [review]
lgtm

**Pull request merged**